WcaOnRails/app/assets/stylesheets/persons.scss

```diff
@@ -29,3 +29,159 @@
     border: 0;
   }
 }
+
+#person {
+  /* Generic table style for the person profile page. */
+  table {
+    @extend table.with-solves;
+    white-space: nowrap;
+
+    .cubing-icon {
+      font-size: 1.3em;
+      line-height: normal;
+      vertical-align: middle;
+    }
+
+    td,
+    th {
+      vertical-align: middle;
+      &.single,
+      &.average {
+        text-align: right;
+      }
+    }
+
+    td {
+      padding: {
+        top: 4px;
+        bottom: 4px;
+      }
+
+      &.single,
+      &.average {
+        font-weight: bold;
+      }
+    }
+  }
+
+  .avatar {
+    max-width: 100%;
+    max-height: 300px;
+  }
+
+  .details {
+    margin-top: 10px;
+
+    th,
+    td {
+      text-align: center;
+
+      .flag {
+        /* A trick to prevent the flag from expanding the row (this effectively lets the flag avoid the impact of td padding). */
+        margin: {
+          top: -10px;
```

Comment:
It's more empirical I'd say. I'm not in love with such solutions, but that's the only way to make things look good that I came up with when playing with this for quite a long time.